src/components/organisms/NavBarSchedule/NavBarSchedule.scss

```
@@ -1,7 +1,7 @@
 @import "scss/constants.scss";
 
-$NavBarSchedule--width: 40%;
-$NavBarSchedule--padding: 20px;
+$navBarSchedule--width: 40%;
+$navBarSchedule--padding: 20px;
```

Same with the uses. Not camel case, no need for double dash as it isn't really a modifier of the same property, they're used for two different props.

```suggestion
$navbar-schedule-width: 40%;
$navbar-schedule-padding: 20px;
```

---

@goran-peoski-work fixed in a57adda4847d34256f8266e362e3a33477f0f3e0